hack/argo-cd/argocd-tls-certs-cm.yaml.tmpl

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-tls-certs-cm
+  labels:
+    app.kubernetes.io/name: argocd-tls-certs-cm
+    app.kubernetes.io/part-of: argocd
+data:
+  'gitea.cnoe.localtest.me': |
+    {{ .SelfSignedCert | indentNewLines 4 }}
+  '{{.Host}}': |
+    {{ .SelfSignedCert | indentNewLines 4 }}
```

Comment:
Why do you include too the same SelSignedCert ? Who is `host` ?

Reply:
When idpbuilder is deployed under a single domain, gitea is exposed under <domain>/gitea. So we need to add CA for the root domain as well.